tools/authors.py

```diff
@@ -97,6 +99,16 @@ def name_key(fullname):
             surname = surname[4:]
         return (surname.lower(), forename.lower())
 
+    # generate set of all new authors
+    if vars(options)['new']:
+        new_authors = authors.difference(all_authors)
+        # Print some empty lines to separate
+        stdout_b.write(("\n\n").encode('utf-8'))
```

Comment:
Could you also add the alphabetic sorting here? I.e. copy lines 112-113.

Reply:
> Could you also add the alphabetic sorting here? I.e. copy lines 112-113.

yep, in latest changes pushed.